azurerm/resource_arm_network_security_rule.go

```diff
@@ -112,15 +112,19 @@ func resourceArmNetworkSecurityRule() *schema.Resource {
 				ConflictsWith: []string{"destination_address_prefix"},
 			},
 
+			//lintignore:S018
 			"source_application_security_group_ids": {
+				// todo: check? but it appears we only take one despite the API allowing multiple
```

Comment:
Should we be renaming this to `source_application_security_group_id` string then?

Reply:
nope - since it's named `id's` in the API so presumably it will support multiple in the future, it just doesn't today (so it's a quick win to remove `MaxItems: 1` when they enable support for that)